.gitignore

```diff
@@ -63,3 +63,6 @@ next-env.d.ts
 .ionide
 
 # End of https://www.toptal.com/developers/gitignore/api/visualstudiocode
+
+# Ignore JetBrains tools
+.idea
```

Comment:
nit. It's fine by me, but I think it's a good practice to keep in `.gitignore` only the project related paths and keep your IDE related paths in your global `.gitignore`

Reply:
Is vscode ok and Idea not? Just joking...